anime-flutter/anime_galaxy/lib/anime_setting/ui/widget/circular_setting/circular_image.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+
+class CircularImage extends StatelessWidget {
+  final double w;
+  final double h;
```

Comment:
use meaningful variable names, you don't see h in the default container properties do you ;)